ubi_manifest/worker/tasks/depsolve.py

```diff
@@ -72,24 +78,59 @@ def depsolve_task(ubi_repo_ids: List[str]) -> None:
             whitelist, debuginfo_whitelist = _filter_whitelist(config)
             blacklist = parse_blacklist_config(config)
 
+            modulelist = config.modules.whitelist
+            mod_dep_map[repo.id] = _make_modular_depsolver_item(
+                client, repo, modulelist, blacklist
+            )
+
             dep_map[repo.id] = _make_depsolver_item(client, repo, whitelist, blacklist)
             debuginfo_dep_map[debuginfo_repo.id] = _make_depsolver_item(
                 client, debuginfo_repo, debuginfo_whitelist, blacklist
             )
+
+        input_repos = set()
```

Comment:
This shouldn't be needed as commented elsewhere, you already have this information in ModularDepsolverItem

Reply:
Removed.